CMakeLists.txt

```diff
@@ -112,8 +115,7 @@ if (BUILD_SUITE)
 	else (PLX_FOUND OR PXI_FOUND)
 		#Find the PLX Library
 	   find_package(PLX REQUIRED)
-   	link_directories(${PLX_LIBRARY_DIR})
-	   #add_definitions("-DPLX_LINUX -DPCI_CODE -DPLX_LITTLE_ENDIAN")
```

Comment:
This line was here as carried over from PixieSuite1. Do we know what it did and that it was safe to remove them?